x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -20,6 +21,21 @@ type BadgeData struct {
 }
 
 func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPayment) (*types.MsgRelayPaymentResponse, error) {
+	if len(msg.LatestBlockReports) > len(msg.Relays) {
+		return nil, utils.LavaFormatError("RelayPayment_invalid_latest_block_reports", fmt.Errorf("invalid latest block reports"),
+			utils.LogAttr("latestBlockReports", msg.LatestBlockReports),
+			utils.LogAttr("len(latestBlockReports)", len(msg.LatestBlockReports)),
+			utils.LogAttr("relays", msg.Relays),
+			utils.LogAttr("len(relays)", len(msg.Relays)),
+		)
+	}
+
+	if !commontypes.ValidateString(msg.GetDescriptionString(), commontypes.DESCRIPTION_RESTRICTIONS, nil) {
```

Comment:
this will fail if the description is nil, we don't want that

Reply:
done. see https://github.com/lavanet/lava/commit/b923474cd40ba656fc06008c7eb36a84a6652409